### PR TITLE
[BPMSPL-632] Add correlation key and last modified date to process in…

### PR DESCRIPTION
**Pull request by rbarriuso**
…stance list

Removed dead code.

### CODE REVIEW

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/i18n/Constants_es.properties [hunk 1]
```diff
@@ -105,7 +105,8 @@ Loading=Cargando...
 VariableValueUpdated=Variable {0} actualizada
 Ok=Aceptar
 Close=Cerrar
-Correlation_Key=Llave de correlación
+Correlation_Key=Código de negocio
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/i18n/Constants_es.properties]: Avoid adding language specific changes by directly editing these files. These are overridden by syncing with translations stored in Zanata.

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListViewImpl.java [hunk 2]
```diff
@@ -15,14 +15,44 @@
  */
 package org.jbpm.workbench.pr.client.editors.instance.list.variables.dash;
 
+import static org.dashbuilder.dataset.filter.FilterFactory.equalsTo;
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListViewImpl.java]: I believe you got a different code style formatter, please check the latest from here: https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/master/ide-configuration

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListViewImpl.java [hunk 3]
```diff
@@ -496,6 +503,24 @@ public String getValue( ProcessInstanceSummary object ) {
 
         return startTimeColumn;
     }
+    
+    private Column initLastModificationDateColumn() {
+        // start time
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListViewImpl.java]: last update/modification time?

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/i18n/Constants.properties [hunk 4]
```diff
@@ -106,6 +106,7 @@ VariableValueUpdated=Variable {0} updated
 Ok=Ok
 Close=Close
 Correlation_Key=Correlation key
+Last_Modification_Date=Last modification date
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/i18n/Constants.properties]: Perhaps for the UI we could display it as _Last update_?

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListPresenter.java [hunk 5]
```diff
@@ -140,54 +140,31 @@ public void getData( final Range visibleRange ) {
         try {
             if ( !isAddingDefaultFilters() ) {
                 final FilterSettings currentTableSettings = dataSetQueryHelper.getCurrentTableSettings();
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListPresenter.java]: @nmirasch could you help reviewing this bit, you're probably more familiar with this code.

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListPresenter.java [hunk 6]
```diff
@@ -321,10 +298,12 @@ private ProcessInstanceSummary createProcessInstanceSummaryFromDataSet( DataSet
                 dataSetQueryHelper.getColumnStringValue( dataSet, COLUMN_PROCESS_VERSION, i ),
                 dataSetQueryHelper.getColumnIntValue( dataSet, COLUMN_STATUS, i ),
                 dataSetQueryHelper.getColumnDateValue( dataSet, COLUMN_START, i ),
+                dataSetQueryHelper.getColumnDateValue( dataSet, COLUMN_END, i ),
                 dataSetQueryHelper.getColumnStringValue( dataSet, COLUMN_IDENTITY, i ),
                 dataSetQueryHelper.getColumnStringValue( dataSet, COLUMN_PROCESS_INSTANCE_DESCRIPTION, i ),
                 dataSetQueryHelper.getColumnStringValue( dataSet, COLUMN_CORRELATION_KEY, i ),
-                dataSetQueryHelper.getColumnLongValue( dataSet, COLUMN_PARENT_PROCESS_INSTANCE_ID, i ) );
+                dataSetQueryHelper.getColumnLongValue( dataSet, COLUMN_PARENT_PROCESS_INSTANCE_ID, i ),
+                dataSetQueryHelper.getColumnDateValue( dataSet, COLUMN_LAST_MODIFICATION_DATE, i ) );
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListPresenter.java]: This mapping could be a good candidate to be isolated and tested to check if the model get the expected values from the dataSetQueryHelper.

**Comment by cristianonicolai** [comment]
ok to test

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListViewImpl.java (hunk 2 above)
Comment (rbarriuso) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListViewImpl.java]: It's strange, I imported the three code style definitions into my Eclipse 4.6.2 and the organize imports styles don't seem to work.

**Comment by rbarriuso** [comment]
I've pushed the requested changes in a single commit

**Comment by cristianonicolai** [comment]
@rbarriuso there still some minor differences in the code style format, ie static imports should come last. Also, would be nice if you could look into adding some unit tests to cover these changes.

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListPresenter.java (hunk 6 above)
Comment (rbarriuso) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListPresenter.java]: Tests added in 3eb2387

**Comment by cristianonicolai** [comment]
ok to test

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
thanks @rbarriuso

**Pull request merged**